### DISCUSSION TITLE
How do I write queries for an unsopported language?

### DISCUSSION THREAD

**Discussion by Muizzyranking** [HiPhish/rainbow-delimiters.nvim#149] [Q&A] [answered]
I am trying to write rainbow-delimeter.scm for solidity for the past one hour and I can not seem to get it to work. How do I write it?

**Comment by HiPhish** [comment, marked as answer]
Have you already read the official [Query Syntax](https://tree-sitter.github.io/tree-sitter/using-parsers#query-syntax) guide and `:h rb-delimiters-custom-query`?

**Reply by Muizzyranking** [reply]
Yes, I did,  but I hae finally been able to figure it out. I was using wrong nodes and I ended up checking how it was written in this repo while checking the syntax tree for the languages and then I eventually saw the pattern. Guess it was skill issue. Thank you.
